<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pinUI</class>
 <widget class="QDialog" name="pinUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>445</width>
    <height>317</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="label">
       <property name="maximumSize">
        <size>
         <width>94</width>
         <height>145</height>
        </size>
       </property>
       <property name="text">
        <string>SYÖTÄ PIN KOODI</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="4" column="2">
        <widget class="QPushButton" name="btnEnter">
         <property name="maximumSize">
          <size>
           <width>137</width>
           <height>24</height>
          </size>
         </property>
         <property name="text">
          <string>ENTER</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="btn1">
         <property name="maximumSize">
          <size>
           <width>137</width>
           <height>24</height>
          </size>
         </property>
         <property name="text">
          <string>1</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="btn2">
         <property name="maximumSize">
          <size>
           <width>137</width>
           <height>24</height>
          </size>
         </property>
         <property name="text">
          <string>2</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QPushButton" name="btn6">
         <property name="maximumSize">
          <size>
           <width>137</width>
           <height>24</height>
          </size>
         </property>
         <property name="text">
          <string>6</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QPushButton" name="btnCancel">
         <property name="maximumSize">
          <size>
           <width>137</width>
           <height>24</height>
          </size>
         </property>
         <property name="text">
          <string>CANCEL</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QPushButton" name="btn9">
         <property name="maximumSize">
          <size>
           <width>137</width>
           <height>24</height>
          </size>
         </property>
         <property name="text">
          <string>9</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="btn3">
         <property name="maximumSize">
          <size>
           <width>137</width>
           <height>24</height>
          </size>
         </property>
         <property name="text">
          <string>3</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QPushButton" name="btn0">
         <property name="maximumSize">
          <size>
           <width>137</width>
           <height>24</height>
          </size>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="btn5">
         <property name="maximumSize">
          <size>
           <width>137</width>
           <height>24</height>
          </size>
         </property>
         <property name="text">
          <string>5</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="3">
        <widget class="QLineEdit" name="userPINfield">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="maximumSize">
          <size>
           <width>423</width>
           <height>24</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="btn4">
         <property name="maximumSize">
          <size>
           <width>137</width>
           <height>24</height>
          </size>
         </property>
         <property name="text">
          <string>4</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QPushButton" name="btn7">
         <property name="maximumSize">
          <size>
           <width>137</width>
           <height>24</height>
          </size>
         </property>
         <property name="text">
          <string>7</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QPushButton" name="btn8">
         <property name="maximumSize">
          <size>
           <width>137</width>
           <height>24</height>
          </size>
         </property>
         <property name="text">
          <string>8</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
